client/src/QAComponents/AnswerModal.jsx

```diff
@@ -30,16 +29,15 @@ const AnswerModal = (props) => {
     const nickname = e.target[1].value
     const email = e.target[2].value
     const id = e.target.parentNode.id
-    console.log('adding answer...', answer, nickname, email, e.target.parentNode.id)
-    axios.post('http://localhost:3000/qa/answer?answer=' + answer + '&nickname=' + nickname + '&email=' + email + '&id=' + id, {
+    const photos = []
+    axios.post('/qa/answer?answer=' + answer + '&nickname=' + nickname + '&email=' + email + '&id=' + id, {
```

Comment:
If we are posting json do we need a query string here? I would also advise changing this to an interpolated string for readability if necessary or passing it as an actual query string value instead of building it into the url.